source/3-landing-page-cicd/cdk/cdk.json

```diff
@@ -7,6 +7,7 @@
     "@aws-cdk/core:stackRelativeExports": "true",
     "github_alias": "<YOUR GITHUB ALIAS>",
     "github_repo_name": "aws-bootstrap-kit-examples",
-    "github_repo_branch": "main"
+    "github_repo_branch": "main",
+    "service_name": "Landing-Page-CICD"
```

Comment:
I think the service_name should still be Landing-Page . CI/CD is a just deployment process